debugging/how_to_test_a_build.adoc

```diff
@@ -0,0 +1,109 @@
+= How to test a build of JBoss Tools components
+
+If you haven't yet gotten your local build installed, please see link:install_a_local_build.adoc[How to Install a Local Build].
+
+== Everything's Installed, Now what? 
+
+Once you have either downloaded a full build of JBoss Tools, installed from an update site, or otherwise installed the pieces you want to test, you can run your eclipse and begin testing. When running in this fashion, there's no easy debugging capability, and so you are basically limited to usability testing and replicating bugs. 
+
+There are, however, three options you have to get more information:
+
+* Checking Logs
+* Tracing
+* Debugging via a second eclipse instance
+
+=== Logs
+
+When using eclipse the main place to look for logging is in the workspace log file located at `<workspace>/.metadata/.log`.
+
+==== How to gather eclipse log and other in JBoss Tools
+
+If you have JBoss Tools installed the simplest way to gather logs and system configuration is to go to: 
+
+   Help > Report Problem
+   
+This will open up a dialog where you can put in a name and description.
+
+When pressing Finish a .zip file is created that you open or share with others (for example in Jira)
+
+=== Tracing
+
+Eclipse comes with a tracing mechanism to allow various messages to be displayed in the console's output based on what tracing flags you pass in. This does require some knowledge of what tracing flags individual plugins expose, and you can usually discover this by browsing through the plugin's source code to see what they expose. 
+
+To enable tracing, two things are required:
+
+* A .options file must live in your eclipse base directory
+* You must run your eclipse with the -debug flag. 
+
+An example of a .options file, for org.jboss.ide.eclipse.as.server.core would look something like:
+
+```
+org.jboss.ide.eclipse.as.core/debug=true
+org.jboss.ide.eclipse.as.core/config=true
+org.jboss.ide.eclipse.as.core/info=true
+org.jboss.ide.eclipse.as.core/warning=true
+org.jboss.ide.eclipse.as.core/severe=true
+org.jboss.ide.eclipse.as.core/finer=true
```

Comment:
The conversation above provides no insight into what I need to change. Therefore, ignoring.